WooCommerce/Classes/POS/Presentation/Orders/PointOfSaleOrderDetailsView.swift

```diff
@@ -1,59 +1,228 @@
 import SwiftUI
+import struct Yosemite.POSOrder
+import struct Yosemite.POSOrderItem
+import struct Yosemite.POSOrderRefund
 
 struct PointOfSaleOrderDetailsView: View {
-    let orderID: String
+    let orderID: String?
     let onBack: () -> Void
-    @Environment(\.horizontalSizeClass) private var horizontalSizeClass
 
-    private var orderTitle: String {
-        switch orderID {
-        case "order1":
-            return "Order 1"
-        case "order2":
-            return "Order 2"
-        default:
-            return "Unknown Order"
-        }
+    @Environment(PointOfSaleOrdersModel.self) private var ordersModel
```

Comment:
I'm curious about the usage of this model by injecting directly into the environment, rather than passing it through the aggregate model. Would you say it breaks the existing design or we should be fine to adopt it? Mainly asking because I tried the same approach for POS settings, but ultimately went through the aggregate model since I needed related services that were already injected.

PD: It could be just because UI is temporary, sorry if that's the case :D